src/test/transaction_tests.cpp

```diff
@@ -438,12 +445,16 @@ BOOST_AUTO_TEST_CASE(test_big_transaction) {
     }
 
     std::vector<Coin> coins;
+    std::vector<CTxOut> spent_outputs;
+    spent_outputs.reserve(mtx.vin.size());
     for (size_t i = 0; i < mtx.vin.size(); i++) {
         CTxOut out;
         out.nValue = 1000 * SATOSHI;
         out.scriptPubKey = scriptPubKey;
+        spent_outputs.push_back(out);
         coins.emplace_back(std::move(out), 1, false);
     }
+    PrecomputedTransactionData txdata(tx, std::move(spent_outputs));
```

Comment:
What's this for?

Reply:
Once we add tx tests using the new BIP341 sighash, txdata will have the correct `spent_outputs`.

Reply:
Better make this correct, or people will be very confused in the future.

Reply:
It just doesn't seem to be used right now. Why not add it later?

Reply:
Ah. Nevermind. I did not see that the stuff above was part of this test.